spec/requests/posts_controllers_spec.rb

```diff
@@ -17,14 +17,7 @@
     it 'renders the index template' do
       get user_posts_path(user)
       expect(response).to render_template(:index)
-    end
-
-    it 'includes correct placeholder text in the response body' do
-      get user_posts_path(user)
-      expect(response.body).to include("Posts by #{user.name}")
-      expect(response.body).to include('Post 1')
-      expect(response.body).to include('Post 2')
-    end
+    end   
```

Comment:
- [ ] @Team Great work fixing all the failing tests, but I noticed you deleted some tests that were part of project requirement in the previous milestone, kindly consider returning the tests, most of them failed because the text you were searching is not in your views page eg  `expect(response.body).to include("Posts by #{user.name}")` will fail because you don't have a text `Posts by #{user.name}` in your posts index page , it would be structured like this `expect(response.body).to include("Name: #{user.name}")` since you have the text `Name: #{user.name}` in your post index view page.